README.md

```diff
@@ -17,6 +19,18 @@ npm start
 ```
 The command prompt `KnowSheet> ` will appear.
 
+... or evaluate one expression from the command-line:
+```bash
+echo '(expression-here)' | npm start
```

Comment:
I thought that introducing an expression here before actually listing available expressions would be confusing.